libraries/helpers.rb

```diff
@@ -20,6 +20,13 @@ def fs_type(mount)
     def fs_remote?(mount)
       fs_type(mount) == 'nfs' ? true : false
     end
+
+    # Validates passed id.
+    #
+    # @return [Numeric, String]
+    def validate_id(id)
+      id.to_i.to_s == id ? id.to_i : id
```

Comment:
@375gnu Is there any reason not to just make this 'id.to_i'?  If the ID is a string this will return the int and if it's an int it'll return the int.  It avoids additional computation on the value with each chef run.